src/Html.php

```diff
@@ -1641,4 +1637,14 @@ public static function escapeJsRegularExpression(string $regexp): string
 
         return $pattern;
     }
+
+    private static function setNameArrayable(string $name): string
+    {
+        return substr($name, -2) !== '[]' ? $name . '[]' : $name;
+    }
+
+    private static function unsetNameArrayable(string $name): string
```

Comment:
It is not setting anything so `set` prefix is confusing.

Reply:
`getNonArrayableName`?

Reply:
Done!